src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailGenerator.java

```diff
@@ -42,12 +42,12 @@ public BlogSummaryMailGenerator() {
     @Autowired
     public BlogSummaryMailGenerator(BlogRepository blogRepository,
```

Comment:
Make class fields `final` then remove hand-crafted constructor and replace it with `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` class level annotation. See the [example](https://github.com/tdziurko/jvm-bloggers/blob/master/src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/AggregatedRssFeedProducer.java#L32).